### ISSUE TITLE
Add overall accuracy (% correct) for Iris dataset as well, for softmax and neural network stats

### ISSUE THREAD

**Issue by aromanro** [aromanro/MachineLearning#11]
As it's currently implemented for MNIST
